### PR TITLE
Authorize/register telegram client

### PR DESCRIPTION
**Pull request by igor04**

### CODE REVIEW

**Comment by ssut** [comment]
Thanks for your great contribution. :) As I can't resolve the merge conflicts right now, because of the personal reason, could you please resolve it instead of me?

File: lib/telegram/auth_props.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  #
+  # Available options:
+  #
+  #   * phone_number: phone number which will be authorized
+  #   * confirmation: proc which should return confirmation code received via
+  #                   text message, or call
+  #   * register(optional): hash with two opetions
+  #     - :first_name: user first name
+  #     - :last_name: user first name
+  class AuthProps
```
Comment (hnatt) [inline lib/telegram/auth_props.rb]: "props" is a different word, not short for "properties".

**Comment by hnatt** [comment]
@igor04 please update Usage section in README with an example and commentary.

**Comment by igor04** [comment]
@ssut, @hnatt updated

**Comment by ssut** [comment]
Thanks for your efforts! Please understand I can't test this PR right now as I've said it before. I, however, am going to merge this PR to the master branch and will release a new version of telegram-rb to the rubygem as possible as I can. ofc I will do the test if this PR works properly before I release it. Thanks again!

**Pull request merged**